functions.php

```diff
@@ -380,45 +380,71 @@ function nightingale_register_required_plugins() {
 	}
 }
 
+// Only perform these over-rides if there are active plugins
+if ($active_plugins) {
```

Comment:
No space after opening parenthesis is prohibitedPHPCS(WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceAfterOpenParenthesis)

Reply:
Fixed